cmd/geras/main.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
-	"github.com/grpc-ecosystem/go-grpc-prometheus"
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
nit: named imports should be grouped with the named imports section below (sadly goimports can't do all that for you...).

Reply:
i'll clean it up then - i only put the name in there because goimports mandated i name it.